nexus/db-model/src/deployment.rs

```diff
@@ -111,6 +111,16 @@ impl From<Blueprint> for nexus_types::deployment::BlueprintMetadata {
     }
 }
 
+/// Abridged form of [`Blueprint`] to get just the
+/// `target_release_minimum_generation` column.
```

Comment:
nit: I can sort of see why you'd want to do this over reading a whole `nexus_types::Blueprint`, but this doesn't seem worth it to me over using `nexus_db_model::Blueprint`.  (Just selecting one column instead of 8 from the same row, right?)  No big deal either way.

Reply:
Don't have a strong opinion here, so used `nexus_db_model::Blueprint`.